tofhir-log-server/src/test/scala/io/tofhir/server/service/ExecutionServiceTest.scala

```diff
@@ -0,0 +1,125 @@
+        .map(executions => {
+          val executionsData = executions._1
+          val count = executions._2
+          count shouldEqual 0
+          executionsData.length shouldEqual count
+          })
+    }
+
+    "should get executions filtered by error status" in {
+      val queryParams = Map("errorStatuses" -> "SUCCESS")
+      executionService.getExecutions("pilot1", "pilot1", queryParams)
+        .map(executions => {
+          val executionsData = executions._1
+          val count = executions._2
+          count shouldEqual 3
+          executionsData.length shouldEqual count
+          (executionsData(1) \ "id").extract[String] shouldEqual "57bb4431-5d58-44d5-b525-a838c0cfa21c"
+          (executionsData(1) \ "errorStatus").extract[String] shouldEqual "SUCCESS"
+          (executionsData(1) \ "startTime").extract[String] shouldEqual "2023-12-01T14:37:31.348+03:00"
+          (executionsData(1) \ "mappingUrls").extract[Seq[String]].length shouldEqual 1
+          (executionsData(1) \ "mappingUrls")(0).extract[String] shouldEqual "https://aiccelerate.eu/fhir/mappings/pilot1/other-observation-mapping"
+        })
+    }
+
+    "should get execution by id" in {
+      executionService.getExecutionById("pilot1", "pilot1", "4df92d34-7712-47a7-8677-15339797c554")
+        .map(execution => {
+          (execution \ "id").extract[String] shouldEqual "4df92d34-7712-47a7-8677-15339797c554"
+          (execution \ "errorStatus").extract[String] shouldEqual "PARTIAL_SUCCESS"
+          (execution \ "mappingUrls").extract[Seq[String]].length shouldEqual 6
+          (execution \ "mappingUrls")(0).extract[String] shouldEqual "https://aiccelerate.eu/fhir/mappings/pilot1/preoperative-risks-mapping"
+        })
+    }
+
+    "should get execution logs by execution id" in {
+      executionService.getExecutionLogs("4df92d34-7712-47a7-8677-15339797c554")
+        .map(logs => {
+          logs.length shouldEqual 11
+
+          // iterate over logs and check project ids/job ids/execution ids are correct
+          (0 to 10).foreach(i => {
+            (logs(i) \ "projectId").extract[String] shouldEqual "pilot1"
+            (logs(i) \ "executionId").extract[String] shouldEqual "4df92d34-7712-47a7-8677-15339797c554"
+            (logs(i) \ "jobId").extract[String] shouldEqual "pilot1"
+          })
+
+          // check the first log -> STARTED
+          (logs.head \ "result").extract[String] shouldEqual "STARTED"
+          (logs.head \ "mappingUrl").extract[String] shouldEqual "https://aiccelerate.eu/fhir/mappings/pilot1/medication-administration-mapping"
+          (logs(1) \ "result").extract[String] shouldEqual "SUCCESS"
+          (logs(1) \ "numOfFhirResources").extract[Int] shouldEqual 5
+          (logs(1) \ "mappingUrl").extract[String] shouldEqual "https://aiccelerate.eu/fhir/mappings/pilot1/medication-administration-mapping"
+
+        })
+    }
+  }
+
+  override def beforeAll(): Unit = {
```

Comment:
Add an application.conf file for test environment and configure the tofhir.logs-path accordingly. Then, remove this redundant beforeAll function.

Reply:
I do not understand, why the function is redundant and how it may be excluded by using a conf file.